components/dictionary.txt

```diff
@@ -82,6 +82,7 @@ DataFrames
 dataset
 deconvolution
 deconvolved
+deidentified
```

Comment:
I'm not actually sure we need this one - according to wiki there is a dash! https://en.wikipedia.org/wiki/De-identification

```suggestion
```